src/components/Sections/GoogleMapsSection/style.css

```diff
@@ -0,0 +1,12 @@
+.google-maps-description-button {
```

Comment:
Is it really necessary to create new class-definitions for this?

Reply:
The main reason for this was that I was not aware that you could style :hover and :focus using tailwind classes.